src/Elliptic/Systems/Xcts/BoundaryConditions/ApparentHorizon.cpp

```diff
@@ -190,8 +190,9 @@ void negative_expansion_quantities(
 
```

Comment:
[optional, random line] there are a lot of divisions by the conformal factor. I'm not sure if it matters because this is a BC, but it may be possible to factor those out and do only one. Probably doesn't matter, though.

Reply:
Haven't profiled this recently, but I'll keep this in mind when doing a round of optimizations 👍 Not much use to do this blindly I think.